Tests/Tests/FBSnapshotTestCase+STPViewControllerLoading.h

```diff
@@ -9,5 +9,18 @@
 #import <FBSnapshotTestCase/FBSnapshotTestCase.h>
 
 @interface FBSnapshotTestCase (STPViewControllerLoading)
-- (UIView *)stp_preparedAndSizedViewForSnapshotTestFromViewController:(UIViewController *)viewController;
```

Comment:
Might be nice to retain this, or a similarly named method, that encompasses both the other helpers (since everything except Pay Context immediately calls both in a row I think?)

Reply:
ah yeah, that's a good call, I'll update